--- ./carbon/templates/album.tpl.org	2006-08-01 11:25:02.056126000 +0200
+++ ./carbon/templates/album.tpl	2006-08-01 11:25:07.556469750 +0200
@@ -126,7 +126,12 @@
                         {g->container type="imageframe.ImageFrame" frame=$theme.params.$frameType
                                       width=$child.thumbnail.width height=$child.thumbnail.height}
                           <a href="{g->url arg1="view=core.ShowItem" arg2="itemId=`$child.id`"}">
+			   {if ($child.entityType == 'GalleryAlbumItem')}
+			    <img  src="themes/carbon/images/agfolder.png" border="0"><br>
+			    <p class="giTitle">{g->text text="%s" arg1=$child.title|markup}</p>
+			   {else}
                             {g->image id="%ID%" item=$child image=$child.thumbnail class="%CLASS% giThumbnail"}
+			   {/if}
                           </a>
                         {/g->container}
                         {elseif isset($child.thumbnail)}
@@ -135,6 +140,7 @@
                           </a>
                         {else}
                           <a href="{g->url arg1="view=core.ShowItem" arg2="itemId=`$child.id`"}" class="giMissingThumbnail">
+				<img src="themes/carbon/images/agfolder.png" border="0"><br>
                             {g->text text="no thumbnail"}
                           </a>
                         {/if}
@@ -176,8 +182,8 @@
                                 showDate=true
                                 showOwner=$showOwner
                                 showSize=true
-                                showViewCount=true
-                                showSummaries=true
+                                showViewCount=false
+                                showSummaries=false
                                 class="giInfo"}
                     </td>
                     {/foreach}
