--- ./carbon/templates/theme.tpl.org	2006-04-06 23:24:18.000000000 +0200
+++ ./carbon/templates/theme.tpl	2006-07-26 14:11:23.202664000 +0200
@@ -22,6 +22,10 @@
 
     {* Include this theme's style sheet *}
     <link rel="stylesheet" type="text/css" href="{g->theme url="theme.css"}"/>
+    <link rel="stylesheet" href='/hoverbox/hoverbox.css' type="text/css" media="screen, projection" />
+    <!--[if IE]>
+    <link rel="stylesheet" href='/hoverbox/ie_fixes.css' type="text/css" media="screen, projection" />
+    <![endif]-->
     <script type="text/javascript" src="{g->url href='themes/carbon/theme.js'}"></script>
   </head>
   <body class="gallery">
--- ./carbon/templates/navigatorMicroThumbs.tpl.org	2006-03-24 21:04:38.000000000 +0100
+++ ./carbon/templates/navigatorMicroThumbs.tpl	2006-07-26 14:14:31.614439000 +0200
@@ -24,17 +24,20 @@
         {assign var="columnIndex" value=0}
       {/if}
 
-      {if ($peer.peerIndex == $data.thisPeerIndex)}
-        <td align="center" width="44" height="40">
-          {g->image item=$peer image=$peer.thumbnail maxSize=40 title="$title"}
-        </td>
-      {else}
-        <td align="center" width="44" height="40">
-          <a href="{g->url arg1="view=core.ShowItem" arg2="itemId=`$peer.id`"}">
-            {g->image item=$peer image=$peer.thumbnail maxSize=40 title="$title"}
-          </a>
-        </td>
-      {/if}
+	{if ($peer.peerIndex == $data.thisPeerIndex)}
+		<td width="44" height="40" align="center" style="background: url(/hoverbox/display.gif) no-repeat;">
+		 {g->image item=$peer image=$peer.thumbnail style="filter:alpha(opacity=33);-moz-opacity:.33;opacity:.33;" maxSize=40 title="Currently Displayed Image"}
+		</td>
+	{else}
+		<td align="center" width="44" height="40">
+		 <div class="hoverbox">
+		  <a href="{g->url arg1="view=core.ShowItem" arg2="itemId=`$peer.id`"}">
+		   {g->image item=$peer image=$peer.thumbnail style="padding-top: 1px;" maxSize=40 title="$title"}
+		   {g->image item=$peer image=$peer.thumbnail class="preview" maxSize=40 title="$title"}
+		  </a>
+		 </div>
+		</td>
+	{/if}
   
       {assign var="lastIndex" value=$peer.peerIndex}
       {assign var="columnIndex" value=$columnIndex+1}
